src/LookingGlassControls.ts

```diff
@@ -287,12 +320,12 @@ export function initLookingGlassControlGUI(lkgCanvas: HTMLCanvasElement) {
 			const dx = -Math.cos(tx) * mx + Math.sin(tx) * Math.sin(ty) * my
 			const dy = -Math.cos(ty) * my
 			const dz = Math.sin(tx) * mx + Math.cos(tx) * Math.sin(ty) * my
-			setTargetX((v) => v + dx * cfg.targetDiam * 0.001)
-			setTargetY((v) => v + dy * cfg.targetDiam * 0.001)
-			setTargetZ((v) => v + dz * cfg.targetDiam * 0.001)
+			cfg.targetX = cfg.targetX + dx * cfg.targetDiam * 0.001
```

Comment:
This change, and other similar ones on this file account for the removal of the UI element tied to these values. Allowing them to still be set by the `OrbitControls` but not refer to a UI element that no longer exists.